cypress/integration/customizer/hfg/hfg-logo-site-identity.spec.js

```diff
@@ -0,0 +1,172 @@
+			expect(req.response.body.success).to.be.true;
+			expect(req.status).to.equal(200);
+			cy.wait(2000);
+			checkFrontEnd();
+		});
+
+	});
+});
+
+function aliasRestRoutes() {
+	let home = Cypress.config().baseUrl;
+	cy.server().
+	route('POST', home + '/wp-admin/admin-ajax.php').
+	as('customizerSave');
+}
+
+function hideTitleAndTagline() {
+	cy.visit('/wp-admin/customize.php');
+
+	cy.get('#accordion-panel-hfg_header').should('be.visible').
+	click();
+
+	//Switches to Logo & Site Identity settings.
+	cy.get(
+		'.hfg--builder-show .hfg--panel-desktop .hfg--row-main .grid-stack-item[title="Logo & Site Identity"]').
+	should('be.visible').
+	click();
+
+	//Hides the site Title and Tagline
+	cy.get(
+		'#inspector-toggle-control-0').
+	click();
+	cy.get(
+		'#inspector-toggle-control-1').
+	click();
+	cy.get('#save').click();
+}
+
+function setupCustomizer() {
+	// Logins to wp and redirect to customizer.
+	cy.visit('/wp-admin/customize.php');
+
+	// Opens customizer panel.
+	cy.get('#accordion-panel-hfg_header').should('be.visible').
+	click();
+
+	// Checks if logo & site identity component is visible and click it.
+	cy.get(
+		'.hfg--builder-show .hfg--panel-desktop .hfg--row-main .grid-stack-item[title="Logo & Site Identity"]').
+	should('be.visible').
+	click();
+
+	//Makes title visible back.
+	cy.get(
+		'#inspector-toggle-control-0').
+	should('be.visible').
+	click();
+
+	//Makes tagline visible back.
+	cy.get(
+		'#inspector-toggle-control-1').
+	should('be.visible').
+	click();
+
+	//Changes blog name
+	cy.get(
+		'#_customize-input-blogname').
+	should('be.visible').
+	click().
+	clear().
+	type('Test blog title').
+	trigger('change');
+
+	//Changes blog description
+	cy.get(
+		'#_customize-input-blogdescription').
+	click().
+	clear().
+	type('Test this description').
+	trigger('change');
+
+	//Switches to Layout settings
+	cy.get('label.logo_component_align.logo_component_padding.logo_component_margin').
+	//should('be.visible').
+	click();
+
+	//Sets the allignment to Center
+	cy.get('#customize-control-logo_component_align > .neve-white-background-control > .neve-radio-icons > [aria-label="Center"]').
+	should('be.visible').
+	click();
+
+	//Sets the Margin to 5px
+	cy.get(
+		'#customize-control-logo_component_margin .top-input').
+	click().
+	clear().
+	type('{uparrow}{uparrow}{uparrow}{uparrow}{uparrow}').
+	trigger('change');
+
+	//Switches to Style settings
+	cy.get('.logo_color').
+	//should('be.visible').
+	click();
+
+	//Clicks on the color change button
+	cy.get('#customize-control-logo_color > .neve-white-background-control > .neve-control-header > .components-dropdown > .components-button').
+	should('be.visible').
+	click();
+
+	//Changes the color
+	cy.get('input.components-text-control__input').
+	click().
+	clear().
+	type(color);
+
+	cy.get('#save').click();
+
+}
+
+function checkFrontEndHided() {
+
+	cy.visit('/');
+	cy.get('a.brand').
+	should('not.be.visible');
+}
```

Comment:
This function could be a Cypress command. Check the doc of Custom Commands: docs.cypress.io/api/cypress-api/custom-commands.html